src/constants.js

```diff
@@ -52,7 +52,7 @@ export const DECODED_ROUTES = {
   ],
 };
 
-export const UPLOAD_FILE_MAX_SIZE = 20 * 1024 * 1024; // 100mb
+export const UPLOAD_FILE_MAX_SIZE = 1024 * 1024 * 1024; // 1GB
```

Comment:
```suggestion
export const UPLOAD_FILE_MAX_SIZE = 20 * 1024 * 1024; // 20 MiB
```

Changing to 1GB is a major change that would need some discussion first.

What I would suggest is changing this to a configuration variable so that it can be set in `.env` to whatever people want for their instance, and leave the default at 20 MiB. Or you can just leave it at 20 MiB for now, and at least it's all using one constant. Or you can post on Slack/Forum and get buy-in on changing the default to 1GB, but I don't know if that would pass.

```ts
const DEFAULT_MAX_UPLOAD_SIZE = 20 * 1024 * 1024; // 20 MiB
export const getMaxUploadFileSize = () => {
    const config = getConfig();
    return config.MAX_FILE_UPLOAD_SIZE ? parseInt(config.MAX_FILE_UPLOAD_SIZE, 10) : DEFAULT_MAX_UPLOAD_SIZE;
}
```